src/org/aigents/nlp/lg/Dictionary.java

```diff
@@ -0,0 +1,59 @@
+package org.aigents.nlp.lg;
+
+import java.util.*;
+import java.io.*;
+
+public class Dictionary {
+	ArrayList<Word> words;
```

Comment:
It can go that way for starters, but you may find that you better use HashSet or TreeSet so you can navigate the items easily. Refer to this http://www.javapractices.com/topic/TopicAction.do?Id=65